src/test/util/uvu-timeout.ts

```diff
@@ -4,9 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import {IS_WINDOWS} from '../../util/windows.js';
+
 import type * as uvu from 'uvu';
 
-const DEFAULT_TIMEOUT = Number(process.env.TEST_TIMEOUT ?? 30_000);
+const DEFAULT_TIMEOUT = Number(
+  process.env.TEST_TIMEOUT ?? (IS_WINDOWS ? 60_000 : 30_000)
+);
```

Comment:
Super nit: I'm guessing Windows specifically needed more time for some of these. Is there harm in just changing the default globally to 60? Or can we set the `TEST_TIMEOUT` environment variable for windows tests? The conditional default value in the code just seems kinda weird.

Reply:
Yeah, one of these needed more time on Windows. I've seen some other tests occasionally time out on windows too, so I figured I'd just do it universally -- but you're right it's probably better to keep the other ones at 30 seconds, since that gives faster results when there are genuine test failures. And we can just add more overrides the next time we see a timeout.